src/js/components/sharedComponents/DatePicker.jsx

```diff
@@ -234,8 +234,7 @@ export default class DatePicker extends React.Component {
                     <a
                         href="#null"
                         onClick={this.toggleDatePicker}
-                        className="usa-da-icon picker-icon date"
-                        aria-haspopup="true">
```

Comment:
Was this throwing an error in axe? It makes sense to me that we would want to indicate to assistive tech that this link opens a popup.

Reply:
yeah, it randomly is in all of my branches, linter error, it will not build with that there.